composer.json

```diff
@@ -16,19 +16,19 @@
             "name":"Michael Bodnarchuk"
         }
     ],
-    "minimum-stability": "RC",
+    "minimum-stability": "dev",
     "require": {
-        "php": ">=7.4.0 <=8.1 | ~8.1.0",
-        "codeception/lib-innerbrowser": "^1.0",
-        "codeception/codeception": "^4.0"
+        "php": ">=8.1.0",
```

Comment:
Is there a compelling reason not to allow `PHP 8.0`?

Reply:
No my mistake. I thought Codeception v5.0 required 8.1 but I can see it only requires 8.